src/js/game/GameController.js

```diff
@@ -1265,12 +1255,14 @@ class GameController {
       if (blockType !== "cropWheat" || this.levelModel.groundPlane.getBlockAt((this.levelModel.player.position)).blockType === "farmlandWet") {
         this.levelModel.player.updateHidingBlock(this.levelModel.player.position);
         this.levelView.playPlaceBlockAnimation(this.levelModel.player.position, this.levelModel.player.facing, blockType, blockTypeAtPosition, () => {
-          let force = false;
           if (this.checkMinecartLevelEndAnimation() && blockType === "rail") {
-            blockType = this.checkRailBlock(blockType);
-            force = true;
+            if (this.levelModel.player.position[1] < 7) {
+              blockType = "railsUnpoweredVertical";
+            } else {
+              blockType = "rails";
+            }
```

Comment:
👍 much cleaner

Would be nice to see a comment explaining the motivation for this oneoff logic